apps/mail/components/context/command-palette-context.tsx

```
@@ -32,6 +30,16 @@ type Props = {
   children?: React.ReactNode | React.ReactNode[];
 };
 
+type CommandItem = {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'CommandItem'. Consider to delete it or rename it.

'CommandItem' is defined here:


(lint/suspicious/noRedeclare)